src/app/featureFlags/reducer.ts

```diff
@@ -0,0 +1,26 @@
+import { Reducer } from 'redux';
+import { getType } from 'typesafe-actions';
+import { receiveExperiments } from '../actions';
+import { AnyAction } from '../types';
+import { experimentIds, experiments } from './constants';
+import { State } from './types';
+
+export const initialState: State = {};
+
+const reducer: Reducer<State, AnyAction> = (state = initialState, action): any => {
+    switch (action.type) {
+        case getType(receiveExperiments):
```

Comment:
can you also reduce receiveFeatureFlags? even if we don't use them for anything i'd be nice to have them and migrate towards using them from here, and having them here for debugging wouldn't suck